quality-assurance/bdd.md

```diff
@@ -67,14 +67,16 @@ Example Mapping sessions can involve the whole team, or just the "Three Amigos"
 
 ### Books
 
-* Discovery: Explore behaviour using examples - Gaspar Nagy
+* Discovery: Explore behaviour using examples - Gáspár Nagy and Seb Rose
```

Comment:
Wouldn't it be better not to have the query string in the URL?  Neater, anyway.

Reply:
Have cleaned the links :-)